### ISSUE TITLE
Could not create Target notebook in One Note

### ISSUE THREAD

**Issue by HariRag** [stefankueng/EvImSync#54]
![image](https://github.com/stefankueng/EvImSync/assets/153352748/8812354d-6853-4290-ba07-645208ad7844)
I am getting this error, which is different from the one already solved. I checked and double checked my filenames; they only have text characters!

I am using Windows 11 Pro (23H2) with One for Windows 10 App  / One Note App / Microsoft One Note 2010 all installed on the Laptop!

**Comment by adamfisher** [comment]
My ENEX file had extra dots in the filename so I removed those and made sure it was less than 10 characters and then it imported successfully.

**Comment by 1ROFL** [comment]
I'm having the same issue. I renamed it one word under 10 characters and it still didn't solve the issue.

**Comment by 1ROFL** [comment]
> ![image](https://private-user-images.githubusercontent.com/153352748/289256779-8812354d-6853-4290-ba07-645208ad7844.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIxNzEzODgsIm5iZiI6MTcwMjE3MTA4OCwicGF0aCI6Ii8xNTMzNTI3NDgvMjg5MjU2Nzc5LTg4MTIzNTRkLTY4NTMtNDI5MC1iYTA3LTY0NTIwOGFkNzg0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMFQwMTE4MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YzNlMGUxMDdlMDVmMjhiODMyNjkxNDUyNWNhYzk4N2YyOGZjMDE0NzBjOGJhNmQzNTdiMjNlYTEzMTY0ZjJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BOrkGw42mtUxpDxY2_LUmzamsGBuaoMvk714xMMIP3U) I am getting this error, which is different from the one already solved. I checked and double checked my filenames; they only have text characters!
> 
> I am using Windows 11 Pro (23H2) with One for Windows 10 App / One Note App / Microsoft One Note 2010 all installed on the Laptop!

Using the desktop version of onenote I went to file --> options --> save & back up and I changed these 
![image](https://github.com/stefankueng/EvImSync/assets/153403975/4753a58f-f1d7-404c-8957-a414baacbefc)

**Comment by HariRag** [comment]
@1ROFL - appreciate your guidance.... what did you change these settings to? Do we need to point this to the folder that has the .enex files?

**Comment by 1ROFL** [comment]
> @1ROFL - appreciate your guidance.... what did you change these settings to? Do we need to point this to the folder that has the .enex files?

I just created a folder on my OneDrive. It wasn’t pointed to my OneDrive before I changed it.

**Comment by andyrainey1** [comment]
Has anyone been able to solve this? I'm having the same issue. Tried changing file paths for Save and Backup, but that hasn't helped.

**Comment by andyrainey1** [comment]
Any suggestions, @stefankueng?

**Comment by stefankueng** [comment]
anyone tried to manually create a notebook with the name first?

**Comment by andyrainey1** [comment]
> anyone tried to manually create a notebook with the name first?

Yes, I tried that as well, and tried renaming the ENEX file to something very basic and creating the new notebook prior to import as well.

**Comment by stefankueng** [comment]
this error usually only happens when using the onenote version that's installed via the windows store, not the real desktop app.

Either install the real onenote application, or make sure there's an "unfiled notes" section.

**Comment by HaseebMusawir** [comment]
Even that error is present when i used the One note desktop version. 

![image](https://github.com/stefankueng/EvImSync/assets/57661694/d31ffdbe-545d-4d50-a761-0321f240f292)

**Comment by stefankueng** [comment]
error 0x80040155 usually happens when an application is not installed properly.

**Comment by Tone2608** [comment]
I am getting the same error as above and I just installed OneNote from Onenote.com. Maybe a new version is causing the problem?
![2023-12-16 18_07_42-](https://github.com/stefankueng/EvImSync/assets/50729277/166d822a-2c95-42bb-9768-5d543536d06f)

**Comment by Tone2608** [comment]
Any thoughts on this @stefankueng ? Seems like several people are affected by this, so not likely operator error. Thank you!

**Comment by AtlAhdaa** [comment]
I too are getting the same error msg.  
![ever](https://github.com/stefankueng/EvImSync/assets/155862449/6daba9e1-dc5f-4d74-903b-e8f6a1ef54c5)

So frustrating!

**Comment by SirSparkles** [comment]
See #65

**Comment by Tone2608** [comment]
It sure looks like the author @stefankueng may have abandoned this utility... Several weeks and no response :-(

**Comment by SirSparkles** [comment]
I don't think so, comits are still fairly frequent: https://github.com/stefankueng/EvImSync/commits/main/ The guy provded a free tool to the world a decade ago and is still supporting it!

Just looks like it was designed to work with the prior version of OneNote and is probably a heap of work to get it working with the newer versions.

**Comment by Tone2608** [comment]
Of course we are all grateful for the free tool, but I would have thought a quick update stating what you implied would be an easy way for the author to let us all know the gravity of the change needed to repair the issue and that patience is needed. I have since moved to a different note app due to the lack of tools to move Evernote to OneNote, so it's no longer an issue for me.

**Comment by stefankueng** [comment]
I monitor the issues and all the comments regularly.
Problem is that this error does not seem to be due to specific notes nor due to some bug in E2O.
I had this problem once, but killing all onenote processes solved the problem. So I don't think the problem is on my side.

Anyway: I haven't seen any comment in all the identical issues here that lead me to believe there's something new I can try to solve the problem.

![obligatory xkcd](https://imgs.xkcd.com/comics/bug_thread.png)

**Comment by yihchuang** [comment]
see the https://github.com/stefankueng/EvImSync/issues/42#issuecomment-2048566846 posted in https://github.com/stefankueng/EvImSync/issues/42
